lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -2334,25 +2344,28 @@ int s1ap_mme_handle_enb_reset(
                      enb_association->sctp_assoc_id, enb_ue_s1ap_id)) != NULL) {
               enb_ue_s1ap_id &= ENB_UE_S1AP_ID_MASK;
               reset_req->ue_to_reset_list[i].enb_ue_s1ap_id = enb_ue_s1ap_id;
-              reset_req->ue_to_reset_list[i].mme_ue_s1ap_id =
-                  ue_ref_p->mme_ue_s1ap_id;
             } else {
               OAILOG_ERROR_UE(
                   LOG_S1AP, imsi64,
                   "Partial Reset Request without any valid S1 signaling "
-                  "connection.Ignoring it \n");
-              // TBD - Here MME should send Error Indication as it is abnormal
-              // scenario.
-              OAILOG_FUNC_RETURN(LOG_S1AP, RETURNerror);
+                  "connection.Sending Reset Ack with recevied signalling "
+                  "connection IDs \n");
+              reset_req->ue_to_reset_list[i].enb_ue_s1ap_id =
+                  (enb_ue_s1ap_id_t) * (s1_sig_conn_id_p->eNB_UE_S1AP_ID);
             }
+            reset_req->ue_to_reset_list[i].mme_ue_s1ap_id =
+                INVALID_MME_UE_S1AP_ID;
+            free_wrapper((void**) &s1_sig_conn_id_p->eNB_UE_S1AP_ID);
           } else {
             OAILOG_ERROR_UE(
                 LOG_S1AP, imsi64,
                 "Partial Reset Request without any valid S1 signaling "
-                "connection.Ignoring it \n");
-            // TBD - Here MME should send Error Indication as it is abnormal
-            // scenario.
-            OAILOG_FUNC_RETURN(LOG_S1AP, RETURNerror);
+                "connection.Sending Reset Ack with recevied signalling "
```

Comment:
The log message says "Sending Reset Ack" but the request is just forwarded to TASK_MME_APP, so no ACK is triggered here. Please revise the log message.

Reply:
The reason I put "Sending Reset Ack" is because, once  this iiti reaches to mme_app task, mme_app sends itti, Reset Ack with same ue_s1ap_id values to s1ap task. Eventually s1ap sends Reset-Ack with received values from mme_app task.